.travis.yml

```diff
@@ -10,39 +11,47 @@ cache:
     - $HOME/.composer/cache/files
 
 env:
-  - TRANSPORT=jackrabbit PACKAGE_VERSION=high
-  - TRANSPORT=doctrine_dbal PACKAGE_VERSION=high
+  global:
+    # on PHP 7.2 we get deprecations about using each. they supposedly come from phpunit 5 itself. TODO: update to phpunit 6 and see if this goes away.
```

Comment:
Maybe you already know this but `each` is deprecated in PHP 7.2:

> **Warning** This function has been DEPRECATED as of PHP 7.2.0. Relying on this function is highly discouraged.

Source: http://php.net/manual/en/function.each.php

So I don't think that the problem comes from PHPUnit. :wink:

Reply:
thanks for pointing that out, but i know that PHP 7.2 deprecated `each`. i hunted down the each-es in this library and the dependencies, but then was left with warnings for each without indication of where it happens. and somebody suggested it could be phpunit itself. now that i am upgrading to phpunit 6, the warning goes away indeed, so i think phpunit 4 and 5 use `each` internally somewhere.